libs/core/src/main/java/com/sourcefuse/jarc/core/softdelete/SoftDeletesRepository.java

```diff
@@ -0,0 +1,50 @@
+package com.sourcefuse.jarc.core.softdelete;
+
+import com.sourcefuse.jarc.core.models.base.SoftDeleteEntity;
+import java.io.Serializable;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.repository.NoRepositoryBean;
+
+@NoRepositoryBean
+public interface SoftDeletesRepository<
+  T extends SoftDeleteEntity, ID extends Serializable
+>
+  extends JpaRepository<T, ID> {
+  List<T> findAllActive();
```

Comment:
We should override existing findAll functions. Not create new ones.

Reply:
Made changes as per discussion